doc/authoring_command_modules/README.md

```diff
@@ -20,64 +20,35 @@ The document provides instructions and guidelines on how to author command modul
 <a name="heading_set_up"></a>Set Up
 ------
 
-Create you dev environment if you haven't already. This is how to do that.  
+Create your dev environment if you haven't already. This is how to do that.  
 
 Clone the repo, enter the repo directory then create your virtual environment.  
 
 For example:
 ```
 git clone https://github.com/Azure/azure-cli.git
-cd azure-cli
-virtualenv env
+git clone https://github.com/Azure/azure-cli-extensions.git
+python -m venv env
 source env/bin/activate
```

Comment:
You assume it is Linux environment.

Reply:
Yes. It is a simple example and for more information about setting up environment, link to https://github.com/Azure/azure-cli-dev-tools#setting-up-your-development-environment for details.

Reply:
Can you add a hint "This is script for Linux"?